spec.yaml

```diff
@@ -326,3 +341,57 @@ definitions:
         example:
           message: "Recipient accepted the share"
           sharedSecret: "hfiuhworzwnur98d3wjiwhr"
+  AcceptedInvite:
```

Comment:
Where is this used?

Reply:
Interesting, this is a left over from the previous PR. I'll drop it.